server/ctrlsubsonic/handlers_cache.go

```diff
@@ -0,0 +1,155 @@
+package ctrlsubsonic
+
+import (
+	"net/http"
+	"path"
+	"fmt"
+
+	"io"
+	"os"
+	"os/exec"
+
+	"github.com/cespare/xxhash"
+)
+
+type encoderProfile struct {
+	format string
+	bitrate string
+	ffmpegOptions []string
+	forceRG bool
+}
+
+var (
+	ENC_PROFILES = map[string]*encoderProfile {
+		"mp3"    : {  "mp3", "128k", []string{"-c:a", "libmp3lame"}                    , false },
+		"mp3_rg" : {  "mp3", "128k", []string{"-c:a", "libmp3lame"}                    , true  },
+		"opus"   : { "opus",  "96k", []string{"-c:a", "libopus", "-vbr", "constrained"}, false },
+		"opus_rg": { "opus",  "96k", []string{"-c:a", "libopus", "-vbr", "constrained"}, true  },
+	}
+	BUF_LEN = 4096
+)
+
+func StreamTrack(w http.ResponseWriter, r *http.Request, trackPath string, client string, cachePath string) {
```

Comment:
i think this can be unexported since it's only used in this package (as well as other funcs in this file)

Reply:
Should be fixed now (if I got the "exported/unexported" idea right). :)